extensions/msal-node-extensions/package.json

```diff
@@ -52,6 +52,7 @@
   },
   "dependencies": {
     "@azure/msal-common": "^9.0.1",
+    "@azure/msal-node-runtime": "file:azure-msal-node-runtime-1.0.0-alpha.0.tgz",
```

Comment:
Temporary - will be updated once the first version of the interop library is released to npm

Reply:
Is this still current?  Wondering if there's a more recent version available you should be integrating with.

Reply:
Updated!